src/include/migraphx/generic_float.hpp

```diff
@@ -320,7 +320,7 @@ struct __attribute__((packed, may_alias)) generic_float
         return *this;
     }
 
-    constexpr generic_float operator++(int) noexcept
+    const generic_float operator++(int) noexcept
```

Comment:
It looks like that produces another tidy warning. We should probably do what we did before and suppress the warning in cmake.